### PR TITLE
Fixed typo in README

### PR DESCRIPTION
**Pull request by ZachHoefler**
Changed "ckeck" to "check"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Select the game object in the **Hierarchy** pane and fill in some additional inf
 * **Server URL** - if you have your own server instance, please type in its url. <span style="color: red">In most cases this field should be left blank.</span>
```
Comment (newkoy191) [inline README.md]: codetoconnect

File: README.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ Select the game object in the **Hierarchy** pane and fill in some additional inf
...
 * **Exception Logging** - by checking this option you will get more precise information about exceptions in your Unity scripts
```
Comment (newkoy191) [inline README.md]: codetoconnect

File: README.md [hunk 3]
```diff
@@ -39,7 +39,7 @@ Select the game object in the **Hierarchy** pane and fill in some additional inf
...
 * **Auto Upload** -  this option defines if the crash reporting feature should send crash reportings automatically without asking the user.
```
Comment (newkoy191) [inline README.md]: codetoconnect

File: README.md [hunk 4]
```diff
@@ -39,7 +39,7 @@ Select the game object in the **Hierarchy** pane and fill in some additional inf
 * **Server URL** - if you have your own server instance, please type in its url. <span style="color: red">In most cases this field should be left blank.</span>
 * **Exception Logging** - by checking this option you will get more precise information about exceptions in your Unity scripts
 * **Auto Upload** -  this option defines if the crash reporting feature should send crash reportings automatically without asking the user. 
-* **Update Manager** - ckeck this option if users should be informed about app updates from inside your app
```
Comment (newkoy191) [inline README.md]: codetoconnect

File: README.md (hunk 3 above)
Comment (newkoy191) [inline README.md]: codetoconnect

File: README.md [hunk 5]
```diff
@@ -39,7 +39,7 @@ Select the game object in the **Hierarchy** pane and fill in some additional inf
...
+* **Update Manager** - check this option if users should be informed about app updates from inside your app
```
Comment (newkoy191) [inline README.md]: codetoconnect

File: README.md (hunk 1 above)
Comment (newkoy191) [inline README.md]: codetoconnect

File: README.md [hunk 6]
```diff
@@ -39,7 +39,7 @@ Select the game object in the **Hierarchy** pane and fill in some additional inf
...
 
 ![alt text](Documentation/04_script_vars.png "Configure script")
```
Comment (newkoy191) [inline README.md]: codetoconnect

**Comment by lumaxis** [comment]
Are you alright @newkoy191? 🤔

**Pull request merged**